hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -158,18 +158,20 @@ private void ignoreFailedTasks(List<ReconOmTask.TaskResult> failedTasks) {
   }
 
   @Override
-  public synchronized void reInitializeTasks(ReconOMMetadataManager omMetadataManager) {
+  public synchronized void reInitializeTasks(ReconOMMetadataManager omMetadataManager,
+                                             Map<String, ReconOmTask> reconOmTaskMap) {
```

Comment:
Could you please add a java doc for this method? It'd be good to know that happens if `reconOmTaskMap` is null, etc.